packages/libtirpc.rb

```diff
@@ -29,8 +29,8 @@ def self.build
 
   def self.install
     system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
-    system "mkdir #{CREW_DEST_DIR}/usr/local/include/rpc"
-    system "mv #{CREW_DEST_DIR}/usr/local/include/tirpc/rpc/* #{CREW_DEST_DIR}/usr/local/include/rpc/"
-    system "curl -Lo #{CREW_DEST_DIR}/usr/local/include/netconfig.h https://ftp.netbsd.org/pub/NetBSD/NetBSD-current/src/include/netconfig.h"
+    system "mkdir #{CREW_DEST_PREFIX}/include/rpc"
+    system "mv #{CREW_DEST_PREFIX}/include/tirpc/rpc/* #{CREW_DEST_PREFIX}/include/rpc/"
+    system "curl -Lo #{CREW_DEST_PREFIX}/include/netconfig.h https://ftp.netbsd.org/pub/NetBSD/NetBSD-current/src/include/netconfig.h"
```

Comment:
I'm wondering if we should include a sha256sum on this file but we can save that for another pull request.